pkg/controllers/cspc-controller/sync.go

```diff
@@ -56,6 +56,12 @@ func (c *Controller) sync(cspc *cstor.CStorPoolCluster, cspiList *cstor.CStorPoo
 		return nil
 	}
 
+	cspcGot, err = c.populateDesiredInstances(cspcGot)
```

Comment:
Can we do update desiredInsstance as part of `UpdateStatusEventually` ?

Reply:
This is done here to make sure that we process cspc only after doing the required mutation.
This code will go away once the mutation webhook is implemented.
Making it part of `UpdateStatusEventually` delays the desired count value to have appeared on the cspc.

Reply:
We alreay doing it in `UpdateStatusEventually` then do we need to do it here again ?